src/js/components/Carousel/stories/Multiple.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Grommet, Box, Carousel, Image } from 'grommet';
+
+const CarouselMulti = () => {
+  const data = [
+    '//v2.grommet.io/assets/Wilderpeople_Ricky.jpg',
+    '//v2.grommet.io/assets/IMG_4245.jpg',
+    '//v2.grommet.io/assets/IMG_4210.jpg',
+    'https://avatars1.githubusercontent.com/u/14203820?s=280&v=4',
+    'https://developer.hpe.com/img/hpe-dev-grommet-gremlin-rockin-static.svg',
+    '//v2.grommet.io/assets/Wilderpeople_Ricky.jpg',
+    '//v2.grommet.io/assets/IMG_4245.jpg',
+    '//v2.grommet.io/assets/IMG_4210.jpg',
+    '//v2.grommet.io/assets/Wilderpeople_Ricky.jpg',
+  ];
+
+  // 3 Carousel Views with up to 3 images in each view
+  const getCarouselView = () => {
+    const carouselView = new Array(3)
+      .fill([])
+      .map((_, i) => data.slice(i * 3, i * 3 + 3));
+
+    // add images to each view
+    return carouselView.map(view => {
+      // return each view with three images
+      return (
+        <Box direction="row">
+          {view.map(img => (
+            <Image src={img} fit="contain" />
+          ))}
+        </Box>
+      );
+    });
+  };
+  return (
+    <Grommet>
+      <Carousel controls="arrows">{getCarouselView()}</Carousel>
```

Comment:
Shouldn't the function just be a functional component?
So something like:
```
<Carousel controls="arrows"><View /></Carousel>
```
Instead of calling a function?

Reply:
Not Sure if I was missing something but when I tried to use it as a functional component I wasn't able to get the carousel to actually work with just 3 images, instead all the images were rendered 
<img width="1579" alt="Screen Shot 2020-08-12 at 2 17 47 PM" src="https://user-images.githubusercontent.com/49867133/90052666-780d2d00-dca7-11ea-837c-bf91c2b9e9a8.png">